packages/core/src/components/Tooltip/index.tsx

```diff
@@ -194,7 +203,11 @@ export class Tooltip extends React.Component<TooltipProps & WithStylesProps, Too
 
   private handleMouseEnter = () => {
     if (!this.props.toggleOnClick) {
-      this.handleOpen();
+      if (this.props.popover) {
+        this.delaySetPopupVisible(true, MOUSE_ENTER_DELAY);
```

Comment:
I understand the mouse out delay, but I'm not sure I follow why we need the delay in visibility. can we remove this? esp since it is `0`?